beater/beater.go

```diff
@@ -38,7 +40,22 @@ func (bt *beater) Run(b *beat.Beat) error {
 
 	go notifyListening(bt.config, pub.Send)
 
-	bt.server = newServer(bt.config, pub.Send)
+	settings := map[string]interface{}{}
+	if bt.config.Frontend.isEnabled() {
+		cfg := b.Config.Output.Config()
+		esClients, err := elasticsearch.NewElasticsearchClients(cfg)
+		if err != nil {
+			logp.Err(err.Error())
+		}
+		cacheSize := bt.config.Frontend.SourcemapCacheSize
+		smapAccessor, err := utility.NewSourcemapAccessor(esClients, cacheSize)
```

Comment:
If this fails we have bigger problems.

Reply:
The question here is do we want to let APM fail when there is an issue with Sourcemapping? In my opinion Sourcemapping is an add-on, that is why I decided to log errors here and not panic.

Reply:
Agreed, here I think it's fine to log and continue.  I expect this failure, basically allocating the cache, to be very rare.